src/themes/shared/typography.ts

```diff
@@ -4,7 +4,7 @@ import { fontWeights } from './fontWeights';
 import { fontSizes } from './fontSizes';
 import { fonts } from './fonts';
 
-export const typography = {
+export const textStyles = {
```

Comment:
After checking the documentation in chakra -> [Text styles](https://chakra-ui.com/docs/styled-system/text-and-layer-styles#text-styles)
We need to rename the key to make easy it use through the components.

Reply:
With this change. The PR release would be a breaking change due to the email templates pkg that is using the key `typography`. Therefore, I will work on the breaking change on the email templates. Thank you!